WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.java

```diff
@@ -1049,4 +1121,113 @@ private void showActiveQuickStartTutorial() {
 
         ((WPMainActivity) getActivity()).showQuickStartSnackBar(shortQuickStartMessage);
     }
+
+    private void showQuickStartDialogTaskPrompt(final boolean showContinueQuickStartDialog) {
+        if (!isAdded() || getView() == null) {
+            return;
+        }
+
+        final QuickStartMySitePrompts mySitePrompt =
+                QuickStartMySitePrompts.getPromptDetailsForTask(getPromptedQuickStartTask());
+
+        String title;
+        String message;
+
+        if (showContinueQuickStartDialog) {
+            title = getString(R.string.quick_start_dialog_continue_setup_title);
+            message = getString(R.string.quick_start_dialog_continue_setup_message);
+        } else if (mySitePrompt != null) {
+            title = getString(mySitePrompt.getPromptDialogTitleId());
+            message = getString(mySitePrompt.getPromptDialogMessageId());
+        } else {
+            // nothing to show
+            return;
+        }
+
+        mQuickStartTaskPromptSnackBar = WPDialogSnackbar.make(getActivity().findViewById(R.id.coordinator),
```

Comment:
`WPDialogSnackbar` methods can be chained together like `Snackbar`.  As an example, these are equivalent.
```
WPDialogSnackbar wpDialogSnackbar = WPDialogSnackbar.make(view, "Message", Snackbar.LENGTH_INDEFINITE);
wpDialogSnackbar.setTitle("Title");
wpDialogSnackbar.setNeutralButton("Neutral", new OnClickListener() {
    @Override public void onClick(View view) {
        Toast.makeText(getActivity(), "Neutral tapped", Toast.LENGTH_SHORT).show();
    }
});
wpDialogSnackbar.setNegativeButton("Negative", new OnClickListener() {
    @Override public void onClick(View view) {
        Toast.makeText(getActivity(), "Negative tapped", Toast.LENGTH_SHORT).show();
    }
});
wpDialogSnackbar.setPositiveButton("Positive", new OnClickListener() {
    @Override public void onClick(View view) {
        Toast.makeText(getActivity(), "Positive tapped", Toast.LENGTH_SHORT).show();
    }
});
wpDialogSnackbar.show();
```
```
WPDialogSnackbar.make(view, "Message", Snackbar.LENGTH_INDEFINITE)
        .setTitle("Title")
        .setNeutralButton("Neutral", new OnClickListener() {
            @Override public void onClick(View view) {
                Toast.makeText(getActivity(), "Neutral tapped", Toast.LENGTH_SHORT).show();
            }
        })
        .setNegativeButton("Negative", new OnClickListener() {
            @Override public void onClick(View view) {
                Toast.makeText(getActivity(), "Negative tapped", Toast.LENGTH_SHORT).show();
            }
        })
        .setPositiveButton("Positive", new OnClickListener() {
            @Override public void onClick(View view) {
                Toast.makeText(getActivity(), "Positive tapped", Toast.LENGTH_SHORT).show();
            }
        })
        .show();
```

That's not required for this, but I wanted to point it out since it's a new custom class/view.

Reply:
Actually, you can like this below, but the way you have is fine as well.  I'm okay with leaving it whichever way you like.
```
        mQuickStartTaskPromptSnackBar = WPDialogSnackbar.make(getActivity().findViewById(R.id.coordinator),
                message, AccessibilityUtils.getSnackbarDuration(getActivity(),
                        getResources().getInteger(R.integer.quick_start_snackbar_duration_ms)))
                .setTitle(title)
                .setPositiveButton(getString(R.string.quick_start_button_positive), new OnClickListener() {
                    @Override public void onClick(View v) {
                        if (shouldDirectUserToContinueQuickStart) {
                            ActivityLauncher.viewQuickStartForResult(getActivity());
                        } else {
                            mActiveTutorialPrompt = mySitePrompt;
                            showActiveQuickStartTutorial();
                        }
                    }
                })
                .setNegativeButton(getString(R.string.quick_start_button_negative), new OnClickListener() {
                    @Override public void onClick(View v) {
                    }
                });

        mQuickStartTaskPromptSnackBar.show();
```